### PR TITLE
Adding Route and links

### PR DESCRIPTION
**Pull request by BBriset**

### CODE REVIEW

**Review by BenZen59 — DISMISSED** [review]

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -1,20 +1,47 @@
...
-      <Menu width="40%" className="menu">
-        <ul>
-          <li className=" menu-item">Watchlist</li>
-        </ul>
-      </Menu>
-
+      <div className="menu">
+        <Menu width="40%">
+          <ul>
+            <li className=" menu-item">Watchlist</li>
```
Comment (Hacene010) [inline src/components/header/Header.jsx]: camelCase plz

File: src/components/header/Header.jsx [hunk 2]
```diff
@@ -1,20 +1,47 @@
...
+          >
+            Keep N
+          </Link>
+        </div>
         <ul className="navlink">
-          <li className="watchpage ">Watchlist</li>
+          <li>
+            <Link
+              to="/FicheFilm"
+              className="watchpage "
```
Comment (Hacene010) [inline src/components/header/Header.jsx]: camelCase

File: src/components/header/Header.jsx [hunk 3]
```diff
@@ -1,20 +1,47 @@
...
+            </li>
+          </ul>
+        </Menu>
+      </div>
       <nav className="navbar ">
-        <div className="logo">Keep N</div>
+        <div className="logo">
+          <Link
+            to="/"
+            className="Home"
```
Comment (Hacene010) [inline src/components/header/Header.jsx]: camelCase

File: src/components/header/Header.jsx [hunk 4]
```diff
@@ -1,20 +1,47 @@
...
-      </Menu>
-
+      <div className="menu">
+        <Menu width="40%">
+          <ul>
+            <li className=" menu-item">Watchlist</li>
+            <li>
+              <Link
+                to="/"
+                className="Home"
```
Comment (Hacene010) [inline src/components/header/Header.jsx]: camelCase

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some changes to do

**Review by BenZen59 — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]
that's better

**Pull request merged**